cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/release.go

```diff
@@ -312,6 +315,51 @@ func (s *Server) helmReleaseFromUnstructured(ctx context.Context, name types.Nam
 	return release, nil
 }
 
+func (s *Server) newRelease(ctx context.Context, packageRef *corev1.AvailablePackageReference, targetName types.NamespacedName) (*corev1.InstalledPackageReference, error) {
+	// HACK: just for now assume HelmRelease CRD will live in the kubeapps namespace
+	kubeappsNamespace := os.Getenv("POD_NAMESPACE")
+	if kubeappsNamespace == "" {
+		// unit tests don't have this env var set
```

Comment:
You can set an env var for the duration of a test - see https://stackoverflow.com/a/66061355 (nice, I didn't know about the new `testing.T.SetEnv`)

Reply:
oh, cool. Will use that, thanks